src/component/PriceCard.jsx

```diff
@@ -0,0 +1,56 @@
+import "../styles/pricecard.css";
+
+function PriceCard() {
+  return (
+    <div className="price-conteiner">
```

Comment:
```suggestion
    <div className="price-container">
```
Typo in the "container" throughout some files.

Reply:
Done.